test.js

```diff
@@ -82,4 +83,21 @@ describe('rollup-plugin-multi-entry', () => {
   describe('with old API', () => {
     setupTests({ useOldAPI: true });
   });
+
+  it('skipped by filter util', () => {
```

Comment:
I don't understand why this test is needed. We should not be explicitly excluding `entry` in such a test, anyway.